tests/basic_test.go

```diff
@@ -202,21 +200,34 @@ routes:
 `)
 
 		const capID0 = "cap0"
+		const capID1 = "cap1"
 		policySpec := types.PolicySpec{
 			Capture: map[string]string{
 				capID0: `routes.running.destination=="0.0.0.0/0"`,
+				capID1: `routes.running.destination=="1.1.1.1/0"`,
 			},
 			DesiredState: stateData,
 		}
-
-		cacheState := types.CachedState{}
+		cacheState := types.CachedState{
+			Capture: map[string]types.CaptureState{
+				capID1: {
+					State: []byte("{}"),
+					MetaInfo: types.MetaInfo{
+						Version:   "333",
+						TimeStamp: time.Now(),
+					},
+				},
+			},
+		}
 		beforeGenerate := time.Now()
 		obtained, err := nmpolicy.GenerateState(
 			policySpec,
 			stateData,
 			cacheState)
 		assert.NoError(t, err)
 		assert.Equal(t, obtained.MetaInfo.TimeStamp, obtained.Cache.Capture[capID0].MetaInfo.TimeStamp)
+		assert.Equal(t, cacheState.Capture[capID1].MetaInfo.TimeStamp, obtained.Cache.Capture[capID1].MetaInfo.TimeStamp)
+		assert.Equal(t, cacheState.Capture[capID1].MetaInfo.Version, obtained.Cache.Capture[capID1].MetaInfo.Version)
```

Comment:
Can this me unified into: `assert.Equal(t, cacheState.Capture[capID1], obtained.Cache.Capture[capID1])` ?